cosmoz-treenode-navigator.html

```diff
@@ -65,29 +70,46 @@
 			}
 		</style>
 		<div id="header">
-			<h3 class="layout horizontal center wrap">
-				<a data-path="" on-click="openNode"><iron-icon icon="home"></iron-icon></a>
+			<h3 class="layout horizontal center wrap">				
+				<paper-icon-button data-path="" on-tap="openNode" icon="home"></paper-icon-button>			
 				<template is="dom-repeat" items="[[ _nodesOnOpenNodePath ]]" as="node">
-					<span class="slash">/</span><a data-path$="[[ node.path ]]" on-click="openNode">[[ _getNodeName(node) ]]</a>
+					<span class="slash">/</span>
+					<span
+						class="pointer"
+						tabindex="0"
+						data-path$="[[ node.path ]]"
+						on-tap="openNode">[[ _getNodeName(node) ]]
+					</span>
 				</template>
 			</h3>
 			<paper-input tabindex="0" id="searchInput" class="flex" label="[[ searchPlaceholder ]]" title$="[[ searchPlaceholder ]]"
-				id="searchInput" value="{{ searchValue }}">
-				<a suffix hidden$="[[ !_search ]]" on-tap="_clearSearch"><iron-icon icon="clear"></iron-icon></a>
+				id="searchInput" value="{{ searchValue }}">				
+				<paper-icon-button icon="clear" suffix hidden$="[[ !_search ]]" on-tap="_clearSearch"></paper-icon-button>
 			</paper-input>
 		</div>
 		<iron-list items="[[dataPlane]]" as="node">
 			<template>
 				<div>
-					<div class="section" hidden$="[[ !_renderSection(_search, index, dataPlane, node) ]]">[[ node.sectionName ]]</div>
+					<div
+						hidden$="[[ !_renderSection(_search, index, dataPlane, node) ]]"
+						class="section">[[ node.sectionName ]]
+					</div>
 					<paper-item on-tap="_nodeSelected" class="nodeItem">
 						<span class="flex">[[ node.name ]]</span>
-						<iron-icon hidden$="[[ !hasChildren(node) ]]" data-path$="[[ node.path ]]" icon="icons:arrow-forward" on-click="openNode"></iron-icon>
+						<iron-icon
+							hidden$="[[ !hasChildren(node) ]]"
+							icon="icons:arrow-forward"
+							data-path$="[[ node.path ]]"
+							on-tap="openNode">
```

Comment:
I think it's because the parent element is also listening to on-tap.

`<paper-item on-tap="_nodeSelected" class="nodeItem">`

I also see now that we should actually get the selected element via iron-list and not via `paper-item on-tap="_nodeSelected"`.

I'll change that and that will also resolve the double tap issue.